GitHub-Org-Management-Policy.md

```diff
@@ -43,23 +43,27 @@ members to the organization when requested by a Working Group or team.
 
 ## Repositories
 
-Any organization member may request the creation of a new repository within the
-Node.js Foundation GitHub Organization by opening an issue in the Node.js admin
-repository. Provided there are no objections from any TSC or CommComm members,
-such requests are approved automatically after 72 hours. If any objection is
-made, the request may be moved to a vote in each of the Technical Steering and
-Community Committees.  A simple majority of each group *rejecting* the creation
-of the repository is required to block creating the repository. Such requests
-must be posted as issues in the Node.js admin repository.
-
 Any repository created under the Node.js GitHub Organization is considered to be
 a project under the ownership of the Node.js Foundation, and thereby subject
 to the Intellectual Property and Governance policies of the Foundation.
 
-No repository may be deleted, transferred into, or transferred out of the
-Node.js Foundation GitHub Organization without a simple majority of both the
-TSC and CommComm in favor of the action. In certain cases, Node.js Foundation
-Board of Directors approval may also be required.
+Any organization member may request the management of repositories within the
+Node.js Foundation GitHub Organization by opening an issue in the
+[Node.js admin repository][nodejs/admin]. The actions requested could be:
+
+- Creating a new repository
+- Deleting an existing repository
+- Archiving an existing repository
+- Transferring a repository into or out of the organization
+
+Provided there are no objections from any TSC or CommComm members raised in
+the issue, such requests are approved automatically after 72 hours. If any
+objection is made, the request may be moved to a vote in each of the
+Technical Steering and Community Committees. Both the TSC and CommComm must
+reject the request for it to be blocked.
```

Comment:
I think we should consider @Trott 's second suggestion as well (https://github.com/nodejs/TSC/pull/495#discussion_r169428036):

> Also, should we consider blocking if *either* TSC or CommComm rejects? That's the way it is now and I think I kind of prefer it that way. Making things a little *easier* to block means you avoid hypothetical situations like this:
> 
> * Someone proposes creating `nodejs/foo`.
> * TSC supports creating repo `nodejs/foo` and CommComm opposes it. It gets created.
> * Someone else proposes deleting `nodejs/foo`
> * TSC opposed deleting the repo and CommComm supports it. It gets deleted.
> * Go to first bullet point. Repeat forever.
> 
> If only one or the other entity is enough to block, then things stay with whatever the status quo is and there's no endless add/delete/add/delete cycle. Admittedly, it's a hypothetical situation that may never occur, but perhaps a worthwhile thought experiment on the importance of deferring to the status quo when there is conflict?

Reply:
Yes, I'd prefer that this sentence...:

> Both the TSC and CommComm must reject the request for it to be blocked.

...be changed to this:

> If either the TSC or CommComm rejects the request, then the request is denied.